### DISCUSSION TITLE
Problem apply_async

### DISCUSSION THREAD

**Discussion by juliGer** [celery/celery#7670] [Q&A]
Hello! I have a problem that when I run apply_async I don't get the task recived in the console, actually sometimes yes and sometimes no I put a print in the tasks.py file of the library and it seems to be running correctly it returns a code that I understand is the one of the task that was generated.

![image](https://user-images.githubusercontent.com/49050724/179848108-823f873b-ef0f-41b5-9643-c5269bd205d0.png)

As you can see in the image below 3 codes were generated from the 3 times I ran epply_async.

![image](https://user-images.githubusercontent.com/49050724/179848258-0b55d3f1-6e0d-4920-b96a-ce6af8394401.png)

One of those 3 was received and executed as you can see in the following image

![image](https://user-images.githubusercontent.com/49050724/179848272-b7584ba5-c33d-4ca0-97be-4360f4384e44.png)

Does anyone know what could be happening? 
Thanks, Greetings!

[a comment by an automated account is not shown]

**Comment by auvipy** [comment]
before opening a bug report please ask your question on the discussion forum first.

**Comment by olii** [comment]
What broker do you use? If it is RMQ did you enable `publisher_confirm`?
